### PR TITLE
OrderedImportsFixer - handle indented groups

### PR DESCRIPTION
**Pull request by pilgerone**
Fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2712

I've tried to keep it small and elegant. It's passing the tests alright, but I hope it meets the standards.

### CODE REVIEW

File: src/Fixer/Import/OrderedImportsFixer.php [hunk 1]
```diff
@@ -368,6 +375,15 @@ private function getNewOrder(array $uses, Tokens $tokens)
 
                                     continue;
                                 }
+                                //if there is any line ending inside the group import, it should be indented properly
```
Comment (julienfalque) [inline src/Fixer/Import/OrderedImportsFixer.php]: Please add an empty line before this comment.

File: src/Fixer/Import/OrderedImportsFixer.php [hunk 2]
```diff
@@ -368,6 +375,15 @@ private function getNewOrder(array $uses, Tokens $tokens)
...
+                                if ('' === $firstIndent &&
```
Comment (julienfalque) [inline src/Fixer/Import/OrderedImportsFixer.php]: Please add an empty line after the opening brace.

File: tests/Fixer/Import/OrderedImportsFixerTest.php [hunk 3]
```diff
@@ -663,16 +663,36 @@ public function provide70Cases()
...
 use function some\a\{fn_a, fn_b, fn_c};
 use A\B;
+use const some\b\{
+    ConstC,
+    ConstA,
+    ConstB
+};
 use const some\a\{ConstA, ConstB, ConstC};
 use some\b\{
-    ClassF,
```
Comment (julienfalque) [inline tests/Fixer/Import/OrderedImportsFixerTest.php]: Why did you change the order here?

Comment (SpacePossum) [inline tests/Fixer/Import/OrderedImportsFixerTest.php]: its an extra test :+1:

Comment (julienfalque) [inline tests/Fixer/Import/OrderedImportsFixerTest.php]: Then it should be a new case instead of updating an existing one.

Comment (SpacePossum) [inline tests/Fixer/Import/OrderedImportsFixerTest.php]: It is a tweak, so it tests more than before. Nothing is lost, as the `expected` part of this change is also tested. Making a new one would duplicate a lot, so why not just fix this?

**Comment by pilgerone** [comment]
Lesson learnt: don't ever rebase before setting the right target.

All sorted out now I guess. Target and base.

File: tests/Fixer/Import/OrderedImportsFixerTest.php (hunk 3 above)
Comment (pilgerone) [inline tests/Fixer/Import/OrderedImportsFixerTest.php]: It's actually a reminiscent of the early stage of my experiments. The tests I've added to the bottom should cover this situation as well. It could be rolled back. But, yes, it doesn't break and it covers a bit more.

At first glance this particular test seems to focus on ordering namespaces vs. functions vs. constants vs. classes but there was already some imports out of order inside the one of the groups. So I'm not really sure of the organization logic there. 

I guess we could refactor the tests as to leave each testing exactly one behavior. I'm not sure this is necessary or desirable, though.

I'm up to whatever you guys decide.

Comment (SpacePossum) [inline tests/Fixer/Import/OrderedImportsFixerTest.php]: I would merge this change in the test in a heartbeat, but that is just my vote ;)

Comment (julienfalque) [inline tests/Fixer/Import/OrderedImportsFixerTest.php]: I actually misread the changes, I'm fine with it :)

File: src/Fixer/Import/OrderedImportsFixer.php [hunk 4]
```diff
@@ -369,6 +376,17 @@ private function getNewOrder(array $uses, Tokens $tokens)
                                     continue;
                                 }
 
+                                //if there is any line ending inside the group import, it should be indented properly
```
Comment (julienfalque) [inline src/Fixer/Import/OrderedImportsFixer.php]: Please add a space after `//`.

**Comment by keradus** [comment]
Thank you @pilgerone.
